sympy/concrete/products.py

```diff
@@ -331,6 +332,66 @@ def _eval_transpose(self):
             return self.func(self.function.transpose(), *self.limits)
         return None
 
+    def is_convergent(self):
+        r"""
+        See docs of Sum.is_convergent() for explanation of convergence
+        in SymPy.
+
+        The infinite product:
+
```

Comment:
This does not look quite right. I would have expected to see the actual infinite product (as on line 352) here, and then something like:

is defined by the sequence of partial products

and then continued more or less as below except that it might not be necessary to repeat the product on line 352. (Instead, you could say, for example, "Then the product converges to a nonzero ...".).